pipeline_steps/common.groovy

```diff
@@ -511,6 +514,8 @@ void clone_repo(String directory, String ssh_key, String repo, String ref, Strin
       git submodule update --init
     """
   }
+  String sha = sh(script: "cd ${directory}; git rev-parse --verify HEAD", returnStdout: true).trim()
+  return sha
```

Comment:
Nit: skip the assignment and just return the result of the `sh()` call.

Reply:
I did it like that to make it easier to work out, at a glance, what `sh` is doing.

Reply:
👍